etna/transforms/decomposition/change_points_based/detrend.py

```diff
@@ -85,3 +106,25 @@ def __init__(
             ),
             required_features=[in_column],
         )
+
+    def params_to_tune(self) -> Dict[str, "BaseDistribution"]:
+        """Get default grid for tuning hyperparameters.
+
+        If ``change_points_model`` isn't set then this grid tunes parameters:
+        ``change_points_model.change_points_model.model``, ``change_points_model.n_bkps``.
+        Other parameters are expected to be set by the user.
+
+        Returns
+        -------
+        :
+            Grid to tune.
+        """
+        if self._is_change_points_model_default:
```

Comment:
This type of solution can cause some problems.

For example, model that was created from `to_dict` won't work like this because for it `self._is_change_points_model_default` is False.

Probably, some other solution can be made to work this around.

Reply:
We want to achieve two things:
- For `to_dict` to create the model that behaves the same as original model. Otherwise, it is a strange behaviour. 
  - The most reliable way is to save unchanged values in `self.change_points_model`: `None`.
- For `params_to_tune` to be able to create a grid for a default value.
  - If we save `self.change_points_model=None` as suggested above then we should somehow create a grid that creates a valid `self.change_points_model`.
  - We can probably achieve this by creating a grid for `change_points_model` consisting of just one value: the default model. But it looks like some kind of hack.
  - We can also probably change the way we check that we can create a grid. For example, by looking at the class we use in `change_points_model`.

Reply:
It can be connected to the problem with `model="catboost"` in #1250.

Reply:
> We can also probably change the way we check that we can create a grid. For example, by looking at the class we use in change_points_model
this^

it's more explicit